backend/editor/settings.py

```diff
@@ -6,6 +6,7 @@
 uri = os.environ.get("NEO4J_URI", "bolt://localhost:7687")
 access_token = os.environ.get("GITHUB_PAT")
 repo_uri = os.environ.get("REPO_URI", "openfoodfacts/openfoodfacts-server")
+frontend_url = os.environ.get("FRONTEND_URL", "http://localhost:3000")
```

Comment:
```suggestion
frontend_url = os.environ.get("FRONTEND_URL", "YOUR_TAXONOMY_EDITOR_URL")
```
If the fallback is used, at least the users would be aware that they have to look for the correct URL (I basically do not like that forgetting to set this environment variable silently fails (same for the other env vars)

Reply:
Even: 
```suggestion
frontend_url = os.environ.get("FRONTEND_URL", "PLEASE_PROVIDE_FRONTEND_URL_ENV")
```